splinepy/io/svg.py

```diff
@@ -0,0 +1,371 @@
+import xml.etree.ElementTree as ET
+from sys import version as python_version
+
+import numpy as np
+from vedo.colors import get_color as _get_color
```

Comment:
you can either do function local import or wrap this with moduleimportraiser

Reply:
used local import